pleskdistup/actions/extensions.py

```diff
@@ -133,3 +133,33 @@ def _revert_action(self) -> action.ActionResult:
 
     def estimate_post_time(self) -> int:
         return 60 * len(self._get_ruby_domains())
+
+
+class UninstallTuxcareEls(action.ActiveAction):
+    """ TuxCare ELS extension is installed on EoL OSes and may enable repositories incompatible with
+        the target OS version. Uninstalling also removes the repositories.
+    """
+
+    def __init__(self):
+        self.name = "uninstall tuxcare-els"
+        self.ext_name = "tuxcare-els"
+
+    def _is_required(self) -> bool:
+        return self.ext_name in dict(plesk.list_installed_extensions())
+
+    def _prepare_action(self) -> action.ActionResult:
+        util.logged_check_call(["/usr/sbin/plesk", "bin", "extension", "--uninstall", self.ext_name])
```

Comment:
NIT: could be moved to the plesk module since we already have list_installed_extensions